pkg/testutil/flags.go

```diff
@@ -0,0 +1,5 @@
+package testutil
+
+import "flag"
+
+var FlagOffline = flag.Bool("testutil.offline", false, "Disable network usage in tests")
```

Comment:
imo `s/testutil.offline/nonet` or `s/testutil.offline/offline/`.

Bikeshed color.  But shorter is better.  And I don't really see the need for dotted prefixes.  We're not a library; we don't need to namespace ourselves in this.

Reply:
Ok, `-offline` it is.